File: src/ElectronicBoard/Contracts/ElectronicBoard.Contracts/Dto/LoginAccountRequest.cs
namespace ElectronicBoard.Contracts.Dto;

public class LoginAccountRequest
{
    /// <summary>
    /// Логин пользователя.
    /// </summary>
    public string Login { get; set; }
    
    /// <summary>
    /// Пароль пользователя.
    /// </summary>
    public string Password { get; set; }
    
}

File: UI/src/app/models/login-account-request.ts
import {StatusUser} from "./filters/userFilter";
import {StatusRole} from "./user";

export interface LoginAccountRequest{
  login: string,
  password: string,
  name: string,
  middleName?: string,
  lastName: string,
  birthday: string,
  phoneNumber: string,
  role: StatusRole,
  photo: "",
  email: string,
  accountId?: number,
  token?: string,
  statusUser:StatusUser,
}
